dolphin/resource_manager/storages.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2013 NetApp
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""The shares api."""
+
+import copy
+
+from os_service_types import exc
+from oslo_log import log
+from oslo_utils import uuidutils, timeutils
+
+from dolphin import db
+from dolphin.db.sqlalchemy import api as db
+from dolphin.driver_manager import manager as drivermanager
+
+LOG = log.getLogger(__name__)
+
+
+def device_model(body):
+    device = DeviceModel()
+    device.id = uuidutils.generate_uuid()
+    device.created_at = timeutils.utcnow()
+    if body.get('name'):
```

Comment:
body is dict..Now rather than going for if condition statements at multiple lines, will it be better to make it one liner as:
device.name = body.get('name', "")
So if you get the value fine, else it will assign the default which is "" here or you can keep it None